llvm/test/tools/llvm-ar/coff-symtab.test

```diff
@@ -14,6 +14,12 @@ RUN: yaml2obj elf.yaml -o coff-symtab.o
 RUN: llvm-ar crs --format coff out3.a coff-symtab.o
 RUN: llvm-nm --print-armap out3.a | FileCheck %s
 
+Create an empty archive with no symbol map, add a COFF file to it.
```

Comment:
It would be useful to know what is expected here, e.g. something like "Show that the output archive is a COFF archive."